src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -5,8 +5,8 @@
 import htsjdk.tribble.util.ParsingUtils;
 import htsjdk.variant.variantcontext.VariantContext;
 import htsjdk.variant.variantcontext.VariantContextBuilder;
-import org.apache.log4j.LogManager;
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
```

Comment:
Not sure why this was necessary, but assuming this is correct.

Reply:
According to @lbergelson that is the correct logger, and using the other logger will not actually work.  I found this to be true in practice - I was missing log messages.